build/installers/scripts/windows/workspace-installer-remover-windows.bat

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
Can you work with Ashley so that these tests can be invoked on the output of the liferay installer build itself?  So we can test the installers as a part of the mvn build itself?

Reply:
These scripts are only for checking after installed our installers, so we can't make it run automatically now. Terry and I will think about it in the future about how to test our installers.